app/helpers/social_image_helper.rb

```diff
@@ -0,0 +1,55 @@
+module SocialImageHelper
+  # After this date we use SocialPreview controller directly rather than passing to URL2PNG.
+  # Keeping old URLs around since they are already generated.
+  SOCIAL_PREVIEW_MIGRATION_DATETIME = DateTime.new(2019, 3, 24)
+
+  def user_social_image_url(user)
```

Comment:
I find the mix of using guard clauses and conditional executions in this method confusing. Did you consider, either:
1. Turning the line 12 to a guard clause so that method looks like this:
```ruby
return GeneratedImage.new(user).social_image unless use_new_social_url?(user)
return user_social_preview_url(user, format: :png) if user.is_a?(User)
return organization_social_preview_url(user, format: :png) if user.is_a?(Organization)
```

or 2. replacing all guard clauses like this:
```ruby
if use_new_social_url?(user)
      user_social_preview_url(user, format: :png) if user.is_a?(User)
      organization_social_preview_url(user, format: :png) if user.is_a?(Organization)
else
     GeneratedImage.new(user).social_image
end